src/clj/doit/todo/db.clj

```diff
@@ -0,0 +1,11 @@
+(ns doit.todo.db
+  (:require [clojure.java.jdbc :as jdbc]
+            [doit.config :as config]))
+
+(defn add-todo!
+  [values]
+  (jdbc/insert! config/db :todo values))
```

Comment:
You can consider `keywordize` the output from the insert so that your tests always have `created-at` keywordized available and there's less confusion the future.

Reply:
`created_at` keyword is already been tested in `db_test`. I think it is okay not to send `created_at` through the API because it doesn't add value to the user.